components/database/couchdb-21/couchdb.p5m

```diff
@@ -0,0 +1,1408 @@
+# to avoid ERROR userland.action001.2        32-bit object 'usr/lib/amd64/couchdb/$(COMPONENT_VERSION_MAJOR)/lib/couch-2.1.1/priv/couchjs' in 64-bit path
+<transform file path=usr/couchdb/$(COMPONENT_VERSION_MAJOR)/lib/(.+)/priv/(.+) -> default pkg.linted true >
+
+<transform file path=usr/couchdb/$(COMPONENT_VERSION_MAJOR)/lib/couch-$(COMPONENT_VERSION)/priv/couch(.+) -> set mode 0555 >
+<transform file path=usr/couchdb/$(COMPONENT_VERSION_MAJOR)/lib/couch-$(COMPONENT_VERSION)/priv/spawnkillable/couch(.+) -> set mode 0555 >
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+depend type=require fmri=runtime/erlang
+depend type=require fmri=runtime/python-27
+depend type=require fmri=library/python/netaddr-27
+depend type=require fmri=database/couchdb-common
+
+link path=usr/couchdb/$(COMPONENT_VERSION_MAJOR)/etc target=../../../../../etc/couchdb/2.1
+
+dir  path=etc/couchdb/$(COMPONENT_VERSION_MAJOR)/local.d owner=couchdb group=couchdb mode=0755
+
+# users should create a link to their preferred log rotating system
+file files/couchdb-21.logrotate   path=etc/couchdb/$(COMPONENT_VERSION_MAJOR)/couchdb-21.logrotate   owner=couchdb group=couchdb mode=0664 preserve=yes
+file files/couchdb-21.logadm.conf path=etc/couchdb/$(COMPONENT_VERSION_MAJOR)/couchdb-21.logadm.conf owner=couchdb group=couchdb mode=0664 preserve=yes
+
+file path=etc/couchdb/$(COMPONENT_VERSION_MAJOR)/vm.args
+file path=etc/couchdb/$(COMPONENT_VERSION_MAJOR)/default.ini
+file path=etc/couchdb/$(COMPONENT_VERSION_MAJOR)/default.d/README
+file path=etc/couchdb/$(COMPONENT_VERSION_MAJOR)/default.d/default.ini
+file path=etc/couchdb/$(COMPONENT_VERSION_MAJOR)/local.ini            owner=couchdb group=couchdb mode=0664 preserve=yes 
+file path=etc/couchdb/$(COMPONENT_VERSION_MAJOR)/local.d/README
+
+link path=usr/bin/couchdb2.1 target=../couchdb/$(COMPONENT_VERSION_MAJOR)/bin/couchdb
+link path=usr/bin/couchjs2.1 target=../couchdb/$(COMPONENT_VERSION_MAJOR)/bin/couchjs 
+link path=usr/bin/couchup2.1 target=../couchdb/$(COMPONENT_VERSION_MAJOR)/bin/couchup 
+
+# mediators
+link path=usr/bin/couchdb target=../couchdb/$(COMPONENT_VERSION_MAJOR)/bin/couchdb \
+     mediator=couchdb mediator-version=2.1
+link path=usr/bin/couchjs target=../couchdb/$(COMPONENT_VERSION_MAJOR)/bin/couchjs \
+     mediator=couchdb mediator-version=2.1
+link path=usr/bin/couchup target=../couchdb/$(COMPONENT_VERSION_MAJOR)/bin/couchup \
+     mediator=couchdb mediator-version=2.1
+
+file path=usr/share/man/man1/couchdb.1 \
```

Comment:
What is this? File should be delivered to /usr/couchdb/2.1/man/man1 or /usr/couchdb/2.1/share/man/man1 and mediated links should be created.